src/entity/aggregator/vdaf_id.rs

```diff
@@ -0,0 +1,157 @@
+use crate::json_newtype;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeSet;
+
+/// https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-vdaf-06#name-iana-considerations-7
+#[derive(Debug, Clone, Copy, Eq)]
+pub enum VdafId {
```

Comment:
Here again, we do have something in `janus_messages` that could work: [`VdafType`](https://docs.rs/janus_messages/latest/janus_messages/taskprov/enum.VdafType.html). As with query type, I'm OK with using this locally defined enum if the `janus_messages` one is unsuitable.

Reply:
This repo also has an enum for vdafs, but this is distinct from that, as it is _only_ the id and not the associated configuration (bins/buckets/length) for any specific instantiation. It's really just the u32